### DISCUSSION TITLE
Configure QR link after nat

### DISCUSSION THREAD

**Discussion by eliecerserna** [snipe/snipe-it#10556] [General]
Hello, I need ideas with the following... I have made an installation following the reference manual and now I have access to my snipe-it inventory server from the local network, I configured an active laptop and a QR has been generated that when scanned It redirects me to the local ip of my server in my .inv file I have this APP_URL=http://192.168.0.10 and it is the ip that appears in the QR when I scan it, they asked me to configure the server again in a datacenter and the provider published the machine to me with the following format: xxxx:5288, my question is how can I configure the public IP assigned to me with the port to appear in the QR code.

**Comment by snipe** [comment]
It shouldn’t require any special configurations - just add the url (with port number) to the `APP_URL` and clear your config cache. 

You’ll also want to purge the existing QR codes as well, as they are cached when generated.

**Reply by snipe** [reply]
(Also, per the docs, the APP_URL should *not* have a trailing slash)

**Reply by eliecerserna** [reply]
Thanks for the reply. at the moment everything works correctly.
